src/Components/Playlist/Playlist.js

```diff
@@ -10,7 +10,7 @@ class Playlist extends Component {
   }
 
   handleNameChange(e) {
-    this.props.onNameChange({ playlistName: e.target.value });
+    this.props.onNameChange(e.target.value);
```

Comment:
Whatever you pass to `this.props.onNameChange` is what you have to deal with in `App.js` … here you were passing an object with a `playlistName` property with the value of the input (`event.target` is the input element), so `updatePlaylistName` was getting `{ playlistName: "whatever was in the input field" }` … that obviously doesn't have a `target` (so it's undefined) and you can't access `value` of undefined